functions.php

```diff
@@ -34,7 +34,7 @@
     );
 }
 
-if ($composer) {
+if (file_exists($composer)) {
```

Comment:
Instead of doing this again, you can save the result of the `file_exists` check above and reuse that value here. Some extra parens wrapping may be required, but that's how I resolved this problem in a theme I'm working on that uses a couple of Sage concepts.